pgconf-nyc24.mdx

```diff
@@ -0,0 +1,62 @@
+
+# Xata’s featured talks
+
+We’re hosting two technical talks at the conference that dive into some of the key challenges and solutions in the Postgres world.
+
+## Postgres schema migrations using the expand/contract pattern
+
+_Monday, 30th September – 15:00 (The Hub)_
+
+_Speaker: Andrew Farries, Software Engineer, Xata_
+
+Schema migrations can be one of the trickiest parts of working with Postgres, especially in large-scale systems where downtime is not an option. In this talk, we’ll introduce the [expand/contract pattern](https://xata.io/blog/pgroll-internals), a proven approach to schema migrations that minimizes risk and ensures backward compatibility throughout the process. Attendees will learn how to:
+
+- Plan and execute schema changes while ensuring the stability of existing functionality.
+- Avoid common pitfalls that can lead to downtime or data inconsistencies.
+- Automate parts of the migration process to simplify future changes.
+
+Andrew will introduce [pgroll](https://xata.io/pgroll), an open-source migration tool for Postgres, developed by Xata that keeps multiple versions of schema live during a migration, and he will show how it can be used to facilitate safe application rollouts across database schema changes.
+If you’re responsible for managing database schema changes in production, this session will provide actionable insights to help you avoid headaches and deploy with confidence. See the [full talk synopsis >](https://postgresql.us/events/pgconfnyc2024/schedule/session/1678-postgres-schema-migrations-using-the-expandcontract-pattern/)
+
+## PostgreSQL operational pain points and solutions
+
+_Tuesday, 1st October – 10:40 (Library)_
+
+_Speakers: [Monica Sarbu](https://twitter.com/monicasarbu), Founder & CEO, Xata | [Tudor Gulubenco](https://twitter.com/tudor_g), CTO, Xata_
+
+Operating a Postgres database, especially at scale, comes with its share of challenges. From performance bottlenecks to schema changes or monitoring, this talk will address some of the most common operational pain points that teams face. More importantly, we’ll be diving into practical solutions to address these issues. In this session, we’ll cover:
+
+- How to diagnose and resolve performance issues using real-world examples.
+- The best practices for [major version upgrades in PostgreSQL](https://xata.io/blog/postgres-major-version-upgrades) environments.
+- Tips around scaling and sharding strategies to stay ahead of potential problems.
+
+Whether you’re running a single Postgres instance or managing a complex cluster, this talk will equip you with the tools and strategies to ensure your database runs smoothly. See the [full talk synopsis >](https://postgresql.us/events/pgconfnyc2024/schedule/session/1849-postgresql-operational-pain-points-and-solutions/)
+
+# Meet our team at the Xata booth
```

Comment:
Not necessary, but it could be nice to have a picture of the Xata team at a booth. And we have a very recent one we could borrow haha

https://x.com/apatheticmagpie/status/1835610736275435711